src/features/rewards/profile/index.tsx

```diff
@@ -97,6 +105,7 @@ export default class Profile extends React.PureComponent<Props, {}> {
             ) : null}
           </StyledTitleWrap>
           {verified && type === 'big' ? (
+            !refreshingPublisher && !publisherRefreshed ?
```

Comment:
Could we split all code bellow (that is inside `StyledContent`) into separate function? This way function will tell us more what is going on, because current code is quire hard to read with nested if statements

Reply:
Done.